eng/emitter-package.json

```diff
@@ -2,16 +2,16 @@
   "name": "typescript-emitter-package",
   "main": "dist/src/index.js",
   "dependencies": {
-    "@azure-tools/typespec-ts": "0.38.2",
+    "@azure-tools/typespec-ts": "0.38.3",
     "@azure-tools/typespec-azure-core": "0.51.0",
     "@azure-tools/typespec-autorest": "0.51.0",
-    "@azure-tools/typespec-client-generator-core": "0.51.1",
+    "@azure-tools/typespec-client-generator-core": "0.51.3",
     "@azure-tools/typespec-azure-resource-manager": "0.51.0",
     "@azure-tools/typespec-azure-rulesets": "0.51.0",
-    "@azure-tools/typespec-liftr-base": "0.7.0",
+    "@azure-tools/typespec-liftr-base": "0.8.0",
```

Comment:
will we upgrade to v0.8 yet?

Reply:
Can you check if the rest api specs repo has upgraded or not? we should follow that.

Reply:
yeah, i checked spec repo is currently v0.7 but has an approved pr to v0.8 https://github.com/Azure/azure-rest-api-specs/pull/32881.